app/flatpak-main.c

```diff
@@ -146,6 +147,7 @@ static GOptionEntry empty_entries[] = {
   { "default-arch", 0, 0, G_OPTION_ARG_NONE, &opt_default_arch, N_("Print default arch and exit"), NULL },
   { "supported-arches", 0, 0, G_OPTION_ARG_NONE, &opt_supported_arches, N_("Print supported arches and exit"), NULL },
   { "gl-drivers", 0, 0, G_OPTION_ARG_NONE, &opt_gl_drivers, N_("Print active gl drivers and exit"), NULL },
+  { "list-installations", 0, 0, G_OPTION_ARG_NONE, &opt_list_installations, N_("Print the paths for system installations and exit"), NULL },
```

Comment:
This docstring should perhaps say "custom installations" ?

To make it clear that this does not include the default system location

Reply:
It shows all the installation directories, including the default system one.

Reply:
oh, I must have misread, then